### ISSUE TITLE
get-sources.sh failed: ogg/vorbis/orc/opus has not been downloaded

### ISSUE THREAD

**Issue by PhilLAL** [mi-g/vdhcoapp#82]
Hi there,
following the BUILD.md instructions, I launched `./get-source.sh` and get a message at the end 
```
ffmpeg has been downloaded
lame has been downloaded
ogg has not been downloaded
vorbis has not been downloaded
opus has not been downloaded
vpx has been downloaded
x264 has been downloaded
xvid has been downloaded
ocamr has been downloaded
voamrwbenc has been downloaded
sdl2 has been downloaded
webp has been downloaded
zlib has been downloaded
jpeg has been downloaded
x265 has been downloaded
orc has not been downloaded
theora has been downloaded
bzip2 has been downloaded
numa has been downloaded
aom has been downloaded
```
ogg, vorbis, orc, and opus have not been downloaded.

Reading output, I found also upside following messages : 
```
HEAD est maintenant sur ea990c7e bug-fix release 3.99.5
Clonage dans 'ogg'...
nc: connection failed, SOCKSv5 error: Connection refused
fatal: Impossible de lire le dépôt distant.

Veuillez vérifier que vous avez les droits d'accès
et que le dépôt existe.
./get-source.sh: ligne 36 : cd: ogg: Aucun fichier ou dossier de ce type
fatal: ni ceci ni aucun de ses répertoires parents (jusqu'au point de montage /) n'est un dépôt git
Arrêt à la limite du système de fichiers (GIT_DISCOVERY_ACROSS_FILESYSTEM n'est pas défini).
```
which litteraly translates as:impossible to read distant repo

The issue is probably the same for git://git.xiph.org/ogg.git & git://git.xiph.org/vorbis.git & git://git.xiph.org/opus.git & git://anonscm.debian.org/pkg-gstreamer/orc.git

**Comment by PhilLAL** [comment]
After some search, it seems some repo changed : 
- ogg : https://gitlab.xiph.org/xiph/ogg.git
- vorbis : https://gitlab.xiph.org/xiph/vorbis.git
- opus : https://gitlab.xiph.org/xiph/opus.git
- orc (see https://anonscm.debian.org/) : https://salsa.debian.org/gstreamer-team/orc.git

**Comment by PhilLAL** [comment]
Sorry, it seems linked to #51  and solved by #73
